core-services/prow/02_config/openshift/route-controller-manager/_prowconfig.yaml

```diff
@@ -0,0 +1,190 @@
+tide:
+  queries:
+  - includedBranches:
+    - community-4.6
```

Comment:
You can remove all the community-4.*, openshift-4, release-4 which are less than 4.12 and keep only community-4.12, openshift-4.12, release-4.12